src/Uno.UI/UI/Xaml/Controls/FocusVisual/SystemFocusVisual.cs

```diff
@@ -128,15 +128,27 @@ private void SetLayoutProperties()
 
 		Visibility = Visibility.Visible;
 
-		var transformToRoot = FocusedElement.TransformToVisual(XamlRoot.VisualTree.RootElement);
-		var point = transformToRoot.TransformPoint(new Windows.Foundation.Point(0, 0));
+		RenderTransform = FocusedElement.RenderTransform;
+		RenderTransformOrigin = FocusedElement.RenderTransformOrigin;
+
+		var parentElement = (UIElement)VisualTreeHelper.GetParent(FocusedElement);
+		var parentTransform = parentElement.TransformToVisual(XamlRoot.VisualTree.RootElement);
```

Comment:
I think the bug is here.
Are there any case if `parentElement` is null?

https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=111379&view=logs&j=00f54b0b-ea95-511b-fcf9-c8a6c6643bcb&t=6068f0e0-13bc-5f2f-2fdd-b1e85c1e59a8&l=2104

Reply:
Indeed. Creating an identity transform would be the best. If the focused element has no parent, it's likely that it's not displayed at all, so the test above should hide the focus visual as well.